### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by Bil0**
* Update some dependencies
* Add script to run tslint
* Add standard-version for release process

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,29 +1,31 @@
 {
   "name": "hyperverse",
-  "version": "1.0.0",
+  "version": "0.0.0",
```
Comment (red-0ne) [inline package.json:3]: Can we keep it at 1 please?

File: package.json [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
-    "pull-stream": "^3.6.12",
+    "pull-stream": "^3.6.13",
     "reflect-metadata": "^0.1.13"
   },
   "devDependencies": {
-    "@types/node": "^12.0.8",
-    "source-map-support": "^0.5.12",
-    "tslint": "^5.17.0",
-    "typescript": "^3.5.2"
+    "@types/node": "^12.6.9",
```
Comment (red-0ne) [inline package.json:25]: I don't think this applies. It should not be node specific for now

File: package.json (hunk 1 above)
Comment (Bil0) [reply package.json:3]: Yes, it'll be in the first *proper* release, with the `tag` and `changeLog`

File: package.json (hunk 2 above)
Comment (Bil0) [reply package.json:25]: Ok, I'll revert this later!

**Review by red-0ne — APPROVED** [review]

**Pull request merged**